### PR TITLE
fix: missing await, missing type Promise

### PR DESCRIPTION
**Pull request by hyungjikim**
Hi, Team.
It seems Promise is missing when defining params type.

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: e71caa0ce5a5e9651597afb4ce2fcbfae020327f

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: e71caa0ce5a5e9651597afb4ce2fcbfae020327f

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: docs/01-app/02-building-your-application/02-data-fetching/04-incremental-static-regeneration.mdx [hunk 1]
```diff
@@ -86,7 +86,8 @@ export async function generateStaticParams() {
 }
 
 export default async function Page({ params }) {
-  const post = await fetch(`https://api.vercel.app/blog/${params.id}`).then(
+  const id = (await params).id
```
Comment (Netail) [inline docs/01-app/02-building-your-application/02-data-fetching/04-incremental-static-regeneration.mdx:89]:
```suggestion
  const { id } = await params
```

**Review by eps1lon — APPROVED** [review]

**Comment by eps1lon** [comment]
Code is not formatted. `pnpm prettier-fix` should fix it.

**Comment by hyungjikim** [comment]
> Code is not formatted. `pnpm prettier-fix` should fix it.

Thank you. Prettier applied.

**Review by ijjk — APPROVED** [review]

**Pull request merged**